docs/intro/tool-fundamentals/packer.md

```diff
@@ -0,0 +1,49 @@
+---
+'title': 'Packer'
+---
+
+# Packer
+
+[Packer](https://www.packer.io) is an open source tool you can use to define _machine images_ (e.g., VM
+images, Docker images) as code. For example, here is how you can use Packer to define an Ubuntu 18.04 Amazon Machine Image (AMI) that has Node.js installed:
```

Comment:
>Amazon Machine Image (AMI)

Should this be a link to AWS definition of AMI?

Reply:
I'm a little torn on this, because we could link _so much_ that our documentation might get overwhelming. But let's give it a try here.

Addressed in bab0783